src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -1029,6 +1059,23 @@ public Boolean setBlocking(Boolean b) {
   public boolean isMute() {
     return mute;
   }
+  
+
+  @Override
+  public String setLocale(String code) {

```

Comment:
These should be marked as abstract and let the subclasses should implement them.

Reply:
No, it should simply set the locale .. unless for some very unlikely reason, it needs to be overridden.

Reply:
not that we really obey it, i'm just going with the java bean convention..  we can leave this as is.